src/main/java/mate/academy/service/AuthenticationServiceImpl.java

```diff
@@ -2,14 +2,18 @@
 
 import mate.academy.exception.AuthenticationException;
 import mate.academy.model.User;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class AuthenticationServiceImpl implements AuthenticationService {
+    private static final Logger logger = LogManager.getLogger(AuthenticationServiceImpl.class);
+
     @Override
     public User login(String login, String password) throws AuthenticationException {
-        //TODO: add corresponding log message about method login was called
+        logger.info("Method login was called. Params: login = {}", login);
         User user = findByLogin(login);
         if (!user.getPassword().equals(password)) {
-            throw new AuthenticationException("Username or password are incorrect");
+            logger.error("Username or password are incorrect");
```

Comment:
You throw exception in main class in try catch block and there you already write error info to log file, so you must not to delete throw new exception and write error again. And in catch block you can write smth like looger.error("....." + e.getMessage());